corehq/apps/accounting/static/accounting/js/pricing_table.js

```diff
@@ -33,82 +73,107 @@ hqDefine('accounting/js/pricing_table', function () {
         };
 
         self.submitDowngrade = function(pricingTable, e) {
-            var finish = function() {
-                if (self.form) {
-                    self.form.submit();
-                }
+            var _submitForm = function() {
+                self.form.submit();
             };
 
             var $button = $(e.currentTarget);
             $button.disableButton();
+
             $.ajax({
                 method: "POST",
-                url: hqImport('hqwebapp/js/initial_page_data').reverse('email_on_downgrade'),
+                url: initialPageData.reverse('email_on_downgrade'),
                 data: {
-                    old_plan: self.currentEdition,
-                    new_plan: self.selected_edition(),
+                    old_plan: self.currentPlan(),
+                    new_plan: self.selectedPlan(),
                     note: $button.closest(".modal").find("textarea").val(),
                 },
-                success: finish,
-                error: finish,
+                success: _submitForm,
+                error: _submitForm,
             });
         };
 
-        self.init = function () {
-            $('.col-edition').click(function () {
-                self.selected_edition($(this).data('edition'));
-            });
+        self.contactSales = function (pricingTable, e) {
+            var $button = $(e.currentTarget);
+            $button.disableButton();
+
+            self.form = $(e.currentTarget).closest("form");
+            self.currentPlan = self.currentPlan + " - annual pricing";
```

Comment:
gettext?

Reply:
nevermind, doesn't look like this is a user-facing string

Reply:
Yeah, this should just be for the email to sales